Update Dutch translation

coreice opened the pull request:
Added over 200 new strings and removed obsolete strings from older (pre 1.3) Bracket versions. Edited some existing strings to be more accurate. Tried to be as consistent as possible when translating certain jargon.

Commit:
Update strings.js

Added over 200 new strings en removed obsolete strings from older Bracket versions. Edited some existing strings to be more accurate. Tried to be as consistent as possible when translating certain jargon.

MarcelGerber commented:
Thanks for your contribution!
You need to sign the [contributor license agreement](http://dev.brackets.io/brackets-contributor-license-agreement.html) first, we can't take your changes otherwise.

abose commented:
@coreice As brackets is an open source project, The  contributor license agreement is for your protection as well as the protection of Brackets and its licensees; it does not change your rights to use your own Contributions for any other purpose.

coreice commented:
@abose I'm aware of that. I hope my contribution will be approved.

abose commented:
@coreice have you signed the CLA?

MarcelGerber commented:
@abose He has (according to `grunt cla-check --user=coreice`), but the Travis build fails either way for some reason. I've restarted it twice now without success.

MarcelGerber commented on src/nls/nl/strings.js:
```diff
@@ -103,63 +129,126 @@ define({
+    "SPLITVIEW_MENU_TOOLTIP"    : "Splits de editor verticaal of horizontaal",
+    "GEAR_MENU_TOOLTIP"         : "Configureer bestanden",
+    
+     "SPLITVIEW_INFO_TITLE"              : "Al geopend",
```
nit: this line is indented one space too much

MarcelGerber commented on src/nls/nl/strings.js:
```diff
@@ -311,41 +434,56 @@ define({
-    "ABOUT_TEXT_WEB_PLATFORM_DOCS"         : "Web Platform Docs en het Web Platform logo zijn gelicentieerd onder een Creative Commons Attribution licentie, <a href='{WEB_PLATFORM_DOCS_LICENSE}'>CC-BY 3.0 Unported</a>.",
-    "UPDATE_NOTIFICATION_TOOLTIP"          : "Er is een nieuwe versie van {APP_NAME} beschikbaar! Klik hier voor details.",
+      "ABOUT_TEXT_WEB_PLATFORM_DOCS"         : "Web Platform Docs en het Web Platform logo zijn gelicentieerd onder een Creative Commons Attribution licentie, <a href='{WEB_PLATFORM_DOCS_LICENSE}'>CC-BY 3.0 Unported</a>.",
+      "UPDATE_NOTIFICATION_TOOLTIP"          : "Er is een nieuwe versie van {APP_NAME} beschikbaar! Klik hier voor details.",
```
nit: remove the extra leading whitespace for these two lines

MarcelGerber commented:
@ispieter @githrdw Would you mind doing a quick review on these changes & additions to the Dutch translation?

abose commented:
@coreice Could you update this PR.

ispieter commented:
Yes, I see several errors in the translations. @abose, what is the best way to mark these? Should I leave a comment on all of them?

coreice commented:
If you can mark these errors I will fix them.

MarcelGerber commented:
@ispieter It's best to add a line note to every line you'd like to be changed.
It'd be very helpful if you could include both an explanation of what's wrong in your opinion and a suggested correction.

@coreice Please make sure you fix the linting errors and the missppelled key, too.

ispieter commented on src/nls/nl/strings.js:
```diff
@@ -103,63 +129,126 @@ define({
+    "EXCLUDE_FILE_FILTER"               : "Sluit {0} uit",
+    "EDIT_FILE_FILTER"                  : "Bewerk\u2026",
+    "FILE_FILTER_DIALOG"                : "Bewerk set uitgesloten bestanden",
+    "FILE_FILTER_INSTRUCTIONS"          : "Sluit bestanden en mappen uit met de volgende stringen/substringen of <a href='{0}' title='{0}'>wildcards</a>. Plaats elke string op een nieuwe regel.",
```
'stringen/substringen' are no Dutch words. Perhaps 'teksten/deelteksten' is a better alternative.

ispieter commented on src/nls/nl/strings.js:
```diff
@@ -251,6 +355,20 @@ define({
+    "CMD_FIND_ALL_AND_SELECT"             : "Zoek alles en selecteer",
+    "CMD_ADD_NEXT_MATCH"                  : "Voeg eerstvolgende overeenkomst toe aan selectie",
+    "CMD_SKIP_CURRENT_MATCH"              : "Sla-over en selecteer volgende overeenkomst",   
+    "CMD_FIND_IN_FILES"                   : "Zoek in mappen",
```
Should be 'Zoek in bestanden' as we're referring to files.

ispieter commented on src/nls/nl/strings.js:
```diff
@@ -265,45 +383,50 @@ define({
     "CMD_TOGGLE_QUICK_DOCS"               : "Snel naar documentatie",
     "CMD_QUICK_EDIT_PREV_MATCH"           : "Vorige overeenkomst",
     "CMD_QUICK_EDIT_NEXT_MATCH"           : "Volgende overeenkomst",
+    "CMD_CSS_QUICK_EDIT_NEW_RULE"         : "Nieuwe regels",
```
Should be 'Nieuwe regel' as 'Rule' is singular.

ispieter commented on src/nls/nl/strings.js:
```diff
@@ -265,45 +383,50 @@ define({
     "CMD_NEXT_DOC"                        : "Volgend document",
     "CMD_PREV_DOC"                        : "Vorig document",
     "CMD_SHOW_IN_TREE"                    : "Toon in bestandsboom",
+    "CMD_SHOW_IN_EXPLORER"                : "Toon in Explorer",
```
'Explorer' could be renamed to 'Verkenner'. Dutch translation of the Windows file manager.

ispieter commented on src/nls/nl/strings.js:
```diff
@@ -311,41 +434,56 @@ define({
+    "INSTALL_EXTENSION_TITLE"              : "Installeer extensie",
+    "UPDATE_EXTENSION_TITLE"               : "Update extensie",
+    "INSTALL_EXTENSION_LABEL"              : "Extensie URL",
+    "INSTALL_EXTENSION_HINT"               : "URL van het zip bestand of de GitHup repo van de extensie",
```
'Githup' should be 'Github'.

ispieter commented on src/nls/nl/strings.js:
```diff
@@ -355,82 +493,128 @@ define({
+    "CHANGE_AND_RELOAD_MESSAGE"              : "Om de aangeduidde extensie te updaten of te verwijderen, moet je {APP_NAME} stoppen en herstarten. Er zal gevraagd worden alle onbewaarde wijzigingen op te slaan.",
+    "REMOVE_AND_RELOAD"                      : "Verwijder extensies en afsluiten",
+    "CHANGE_AND_RELOAD"                      : "Wijzig extensies en afsluiten",
+    "UPDATE_AND_RELOAD"                      : "Update extensies en afsluiten",
```
Line 545, 546 and 547 should use 'herladen' instead of the word 'afsluiten'.

ispieter commented on src/nls/nl/strings.js:
```diff
@@ -355,82 +493,128 @@ define({
+    "REMOVE_AND_RELOAD"                      : "Verwijder extensies en afsluiten",
+    "CHANGE_AND_RELOAD"                      : "Wijzig extensies en afsluiten",
+    "UPDATE_AND_RELOAD"                      : "Update extensies en afsluiten",
+    "PROCESSING_EXTENSIONS"                : "Processing extension changes\u2026",
```
This is not translated yet.

ispieter commented:
These are the things that I found. The rest seems fine to me. :)

coreice commented:
Thanks a lot, I will process them :). I hope it will be ready this week but I've a busy schedule so it also could be next week.

abose commented:
Thanks @ispieter for reviewing this. @coreice Did you get a chance to look into this? Brackets will have a release within the next two weeks.

coreice commented:
@abose Just returned home from a convention. Will be working on it now :). Do you want me to create a new edit and pull request after I'm done?

abose commented:
@coreice :+1: Editing this PR would be good for other reviewers to keep track of the changes.

Commit:
Update strings.js

Fixed some errors reported by @MarcelGerber and @ispieter. Also saw some other errors during the process.

Line 133: Changed "Verwijdering" into "Verwijderen".

Line 251, 485, 486, 500, 521, 522 & 589: These lines weren't (correctly) translated in the previous version of my translation.

coreice commented:
Updated the PR. Thanks for checking @MarcelGerber & @ispieter, there were some stupid mistakes in the previous one. I hope this one will be better :).

MarcelGerber commented:
@ispieter Does this look good to you?

MarcelGerber commented:
You should also update the SHA comment at the very bottom. This is based on 50cbe1b8848786c6be27d4788e4b6a367244abc2, I guess.

Commit:
Update strings.js

Fixed SHA.

coreice commented:
@MarcelGerber You're right. Fixed the SHA comment.

ispieter commented:
Yes, looks good to me.

MarcelGerber commented:
Thank you @coreice @ispieter.
Merging.
